tests/rptest/tests/services_self_test.py

```diff
@@ -230,3 +231,31 @@ def all_partitions_have_segments():
             self.redpanda.nodes,
             lambda n: self.redpanda.start_node(n, first_start=False),
             parallel=True)
+
+
+class CloudSelfTest(Test):
```

Comment:
this can also run in non-cloud contexts, so it can be called it something different

Reply:
i've renamed to `SimpleSelfTest`